vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -309,37 +331,12 @@ ent pod-info
 ``` sh
 ent diag
 ```
-This outputs `kubectl describe` and `kubectl logs` for each of the major Entando pods and can be highly useful when working with Entando Support.
-
-Output:
-```
-ubuntu@entando:~$ ent diag
-Please provide the namespace (entando):
-## DNS rebinding protection TEST
-## LOCAL INFO
-## K8S INFO
-> POD: quickstart-kc-deployer-pbyjdp1dom
->       CONTAINER: deployer
-> POD: quickstart-eci-deployer-smectg3hxy
->       CONTAINER: deployer
-> POD: quickstart-deployer-9ul8cyjtiq
->       CONTAINER: deployer
-> POD: quickstart-composite-app-deployer-nlz9lxc6do
->       CONTAINER: deployer
-> POD: quickstart-eci-k8s-svc-deployment-79c4894767-5p85d
->       CONTAINER: k8s-svc-container
-> POD: quickstart-kc-server-deployment-85987fc84c-flrlw
->       CONTAINER: server-container
-> POD: quickstart-operator-7bfd7fc8cd-gd774
->       CONTAINER: operator
-> POD: quickstart-server-deployment-f69f84798-g6lx5
->       CONTAINER: server-container
->       CONTAINER: de-container
->       CONTAINER: appbuilder-container
+This outputs `kubectl describe` and `kubectl logs` for each of the fundamental Entando pods. It also exports custom resources, ingresses, deployments, "previous" pod logs, namespace events, etc. Diagnostic information can be highly useful when analyzing and troubleshooting behavior. The command `ent diag` outputs the directory paths for diagnostic data, similar to the following
```

Comment:
the last sentence doesn't really make clear that that's where the full diag logs/data can be found

Reply:
it says it outputs the file paths for that info ?? i don't understand. "for diagnostic data" -> "of the diagnostic logs"??